docs/development-tips.md

```diff
@@ -3,27 +3,16 @@
 ### TIP: Loading sibling projects
 Some projects fit within the cfgov-refresh architecture,
 but are not fully incorporated into the project.
-These are known as "non-v1 Django apps."
-In order to visit areas of the site locally where those projects are used,
-the sibling projects need to be installed
-and then indexed within this cfgov-refresh project.
+These are known as "satellite apps."
 
-The non-v1 apps are the following:
+Satellite apps are listed in the
+[optional-public.txt](https://github.com/cfpb/cfgov-refresh/blob/master/requirements/optional-public.txt).
```

Comment:
Extra period after the link.

Reply:
The period is grammatically correct, isn't it?

Reply:
Oh snap, nvm, I see what you mean.